src/Generator/Generators/CLI/CLITypeReferences.cs

```diff
@@ -142,7 +142,13 @@ private void GenerateInclude(ASTRecord<Declaration> record)
         private string GetIncludePath(TranslationUnit translationUnit)
         {
             if (!DriverOptions.UseHeaderDirectories)
-                return translationUnit.FileName;
+                if(DriverOptions.GenerateName != null)
```

Comment:
Fix your indentation - if the indentation is correct, merge the if-s with &&. Add a space after the if.